test/specs/cli.js

```diff
@@ -61,6 +61,12 @@ describe('cli', () => {
     output.code.should.equal(EXIT_CODES.SHX_ERROR);
   });
 
+  it("handles -c 'input' directly from npm scripts", () => {
```

Comment:
I assume the `-c` argument is all our "shell" is required to handle to satisfy the `script-shell` requirement?